### PR TITLE
 US388853 : [iOS] MAS credentials should be stored in the keychain as device only

### PR DESCRIPTION
**Pull request by rodrigoareis**

### CODE REVIEW

File: MASFoundation/Classes/_private_/services/access/MASAccessService.m [hunk 1]
```diff
@@ -102,36 +102,32 @@ - (void)serviceWillStart
...
-    
     _localStorageServiceName = [NSString stringWithFormat:@"%@.%@", _gatewayIdentifier, kMASAccessLocalStorageServiceName];
     
     _sharedStorageServiceName = [NSString stringWithFormat:@"%@.%@", _gatewayIdentifier, kMASAccessSharedStorageServiceName];
     
+    //
+    // Local storage
+    //
+    MASIKeyChainStore *localStorage = [MASIKeyChainStore keyChainStoreWithService:_localStorageServiceName];
+    localStorage.synchronizable = FALSE;
```
Comment (jhoongo) [inline MASFoundation/Classes/_private_/services/access/MASAccessService.m]: Please avoid using TRUE/FALSE, and use YES/NO.

File: MASFoundation/Classes/_private_/services/access/MASAccessService.m [hunk 2]
```diff
@@ -102,36 +102,32 @@ - (void)serviceWillStart
...
-        // Local storage
-        //
-        MASIKeyChainStore *localStorage = [MASIKeyChainStore keyChainStoreWithService:_localStorageServiceName];
-        
         //
         // Shared storage
         //
         MASIKeyChainStore *sharedStorage = [MASIKeyChainStore keyChainStoreWithService:_sharedStorageServiceName accessGroup:self.accessGroup];
-        
+        sharedStorage.synchronizable = FALSE;
```
Comment (jhoongo) [inline MASFoundation/Classes/_private_/services/access/MASAccessService.m]: Please avoid using TRUE/FALSE, and use YES/NO.

**Review by jhoongo — CHANGES_REQUESTED** [review]
Please review my comment, and return the ticket back to me.

**Review by jhoongo — DISMISSED** [review]
looks good to me

**Review by lsanches — APPROVED** [review]
Approved!

**Pull request merged**